edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/BackupAndRestoreTest.cs

```diff
@@ -177,7 +177,6 @@ async Task BackupAndRestoreCorruptBackupDataTest(ITransportSettings[] transportS
                 // Send more messages after the receiver is registered.
                 Task<int> task2 = sender.SendMessagesByCountAsync("output1", beforeBackupMessageCount, afterBackupMessageCount, TimeSpan.FromMinutes(2));
                 sentMessagesCount = await task2;
-                Assert.Equal(afterBackupMessageCount, sentMessagesCount);
```

Comment:
Did this assertion fail because of a timeout while sending the messages? If so, I would rather increase the timeout instead of moving this assertion to after the wait before asserting received messages.

This is so that we can ensure that all the messages were sent before we even attempt to check the receipt of messages later.

Reply:
I thought this was an error caused by messages in flight in the EdgeHubFixture. This is not the case and I am investigating more.

Reply:
I've updated the description with a link to the failing CI run and tested version of BackupAndRestore.cs.